src/MathematicsCaptcha.php

```diff
@@ -0,0 +1,60 @@
+<?php 
+
+namespace ElicDev\MathCaptcha;
+
+use ElicDev\MathCaptcha\MathCaptcha;
+use Illuminate\Session\SessionManager;
+
+class MathematicsCaptcha extends MathCaptcha

```

Comment:
Why do you use a second class extending the first one. 
Why not have just one class "MathCaptcha". But your "operands" array into a config file.